src/runtime_src/core/tools/common/tests/TestSpatialSharingOvd.cpp

```diff
@@ -144,7 +144,12 @@ boost::property_tree::ptree TestSpatialSharingOvd::run(std::shared_ptr<xrt_core:
   try{
     singleHardwareCtxTest.initialize();
   } catch (const std::exception& ex) {
-    logger(ptree, "Error", ex.what());
+    std::string err_msg(ex.what());
+    if(err_msg.find(std::string("0xc01e0009")) != std::string::npos) //context creation failed
```

Comment:
Should this address be hardcoded? Not sure but can this not be different for different platforms? 
IMO we should simply say that "the app failed to create a hardware context, Please make sure not other workload is running on the device" as soon as a hardware context creation fails (an exception is thrown from context creation)

Reply:
Yes, this has to be hardcoded. We are wrapping the driver error into something that is readable for the user. 
No, it won't change for different platforms. Might for different OSes, but we will cross that bridge later.

Do you mean add another try/catch inside the `initialize()` function? I can't cause that function doesn't takes ptree as an arg. Since initialize is already in a try/catch, doesn't makes sense to add another one inside the function!

Reply:
No, I didn't mean adding a new try/catch.
I meant as soon as a hardware context creation fails, instead of doing logger(ptree, "Error", ex.what()); , we should print the message:
"The test failed to create a hardware context, Please make sure not other workload is running on the device"
without any if checks. We can do logger(ptree, "Error", ex.what()); under the verbose check for case if someone really wants to know why context creation failed.

Reply:
Oh I see. Makes sense